scripts/build_lib.sh

```diff
@@ -26,36 +26,33 @@ etcd_build() {
 
   run rm -f "${out}/etcd"
   (
-    cd ./server
```

Comment:
Is this change required for workspace to work or is it a cleanup thanks to usage of workspace? I would recommend to defer changes that are not critical for workspace to work.

Reply:
It's a clean up. I'll revert it.